.gitignore

```diff
@@ -40,6 +40,7 @@ CorePlugins*/
 ProgramData-Server*/
 ProgramData-UI*/
 MediaBrowser.WebDashboard/jellyfin-web/**
+tests/**/launchSettings.json
```

Comment:
This file is generated every time the solution is opened with Visual Studio but it is not a required file for an integration tests project so I think it's safe to ignore it. The tests run fine without this file existing.

Reply:
Figured out why this was being added to the solution and removed this from the .gitignore file